incubator/hnc/pkg/controllers/hierarchy_controller.go

```diff
@@ -84,11 +84,12 @@ type NamespaceSyncer interface {
 
 // Reconcile sets up some basic variables and then calls the business logic.
 func (r *HierarchyReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-	if !ex[req.Namespace] {
-		atomic.AddInt32(&hcTot, 1)
-		atomic.AddInt32(&hcCur, 1)
-		defer atomic.AddInt32(&hcCur, -1)
+	if ex[req.Namespace] {
+		return ctrl.Result{}, nil
 	}
+	atomic.AddInt32(&hcTot, 1)
```

Comment:
Now that we're checking these into the main branch, we should probably give them more reasonable names and maybe group them together. Eg something like:

```
// in stats.go:
type counter int32
func (c *counter) Incr() {
  atomic.AddInt32(c, 1)
}
func (c *counter) Decr() {
  atomic.AddInt32(c, -1)
}
type stats struct {
  hcTotal counter
  hcCurrent counter
  ...
}
var stats stats

// here:
stats.hcCurrent.Incr()
defer stats.hcCurrent.Decr()